app/src/main/java/io/gnosis/safe/ui/settings/app/passcode/EnterPasscodeFragment.kt

```diff
@@ -141,6 +130,12 @@ class EnterPasscodeFragment : BaseViewBindingFragment<FragmentPasscodeBinding>()
                     input.hideSoftKeyboard()
                 }
             }
+            contentView.setOnClickListener {
```

Comment:
having content view as click target is not enough?

Reply:
Probably not. Let me check if the "button" still reacts, if only the background has the listener.